src/vs/workbench/browser/parts/quickinput/quickInput.ts

```diff
@@ -671,12 +681,19 @@ class QuickPick<T extends IQuickPickItem> extends QuickInput implements IQuickPi
 				this.selectedItemsToConfirm = null;
 			}
 		}
+		if (this.validationMessage && this.ui.message.textContent !== this.validationMessage) {
+			this.ui.message.textContent = this.validationMessage;
+			this.ui.inputBox.showDecoration(Severity.Error);
+		} else {
```

Comment:
It looks like the `else` would run when there is a validation message and `this.ui.message.textContent` is already set to it.

Reply:
You are right. That was a bad copy paste from the QuickInput.